### PR TITLE
corrections

### PR DESCRIPTION
**Pull request by MataiasCost**
Fixed the Nano step in the Run3 MC, and including the Run3 data file, only the first two steps (L1 and HLT) work.

The Run3 MC table has also been redone, with the Nano fix

### CODE REVIEW

File: ConditionsConsumed/runCMSDrivers_mc2018.sh [hunk 1]
```diff
@@ -2,8 +2,7 @@
 echo "------------------------------"
 echo "Running step-1/9: GEN"
 echo "------------------------------"
-cmsDriver.py TTbar_13TeV_TuneCUETP8M1_cfi  --conditions 105X_upgrade2018_realistic_v6 -n 5 --era Run2_2018 -s GEN --fileout output_step1_GEN.root --beamspot Realistic50ns13TeVCollision  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)'|& tee output_step1_GEN.log
-
+cmsDriver.py TTbar_13TeV_TuneCUETP8M1_cfi  --conditions auto:run2_mc -n 5 --era Run2_2016 -s GEN --fileout output_step1_GEN.root --beamspot Realistic50ns13TeVCollision  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)'|& tee output_step1_GEN.log
```
Comment (malbouis) [inline ConditionsConsumed/runCMSDrivers_mc2018.sh:5]: I still think the Era here should be `--era Run2_2018`. Why are you setting it as `Run2_2016` ?

File: ConditionsConsumed/runCMSDrivers_mc2022.sh [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+echo "------------------------------"
+cmsDriver.py step8 --runUnscheduled --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s PAT --datatier MINIAODSIM --eventcontent MINIAODSIM --filein file:output_step7_AODSIM.root --fileout output_step8_MINIAODSIM.root --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)' |& tee output_step8_MINIAODSIM.log
+
+
+
+#9 NANOAODSIM
+echo "------------------------------"
+echo "Running step-8/8: NANOAODSIM"
+echo "------------------------------"
+cmsDriver.py step5 --runUnscheduled --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s NANO --datatier NANOAODSIM --eventcontent NANOAODSIM --filein file:output_step8_MINIAODSIM.root --fileout output_step9_NANOAODSIM.root --customise_commands='process.GlobalTag.DumpStat=cms.untracked.bool(True)' |& tee output_step9_NANOAODSIM.log
```
Comment (malbouis) [inline ConditionsConsumed/runCMSDrivers_mc2022.sh:68]: I think it is OK for now, but we really should try to understand why it is not possible to run every single step separately. @MataiasCost , please let's investigate it further although it is not a showstopper for now, imho.

File: ConditionsConsumed/runCMSDrivers_mc2018.sh (hunk 1 above)
Comment (MataiasCost) [reply ConditionsConsumed/runCMSDrivers_mc2018.sh:5]: My mistake, checking the source code:
 https://github.com/ravindkv/MyAlCaTools/blob/master/runCMSDrivers_mc2018.sh
it's possible to see that must be --era Run2_2018. I will fix that

File: ConditionsConsumed/runCMSDrivers_mc2022.sh (hunk 2 above)
Comment (MataiasCost) [reply ConditionsConsumed/runCMSDrivers_mc2022.sh:68]: The AODSIM, GEN and SIM are the only that we cannot run separately, but AODSIM, in the case of 2018, don't run separately too. I can give another try in the case of GEN and SIM.

Source: https://github.com/ravindkv/MyAlCaTools/blob/master/runCMSDrivers_mc2018.sh

Comment (malbouis) [reply ConditionsConsumed/runCMSDrivers_mc2022.sh:68]: No, that's fine. the GEN-SIM step is fine.

**Pull request merged**